src/SFA.DAS.EmployerIncentives.Events/IncentiveApplications/SubmittedHandler.cs

```diff
@@ -20,10 +23,25 @@ public Task Handle(Submitted @event, CancellationToken cancellationToken = defau
         {
             var command = new CreateIncentiveCommand(
                 @event.AccountId,
-                @event.IncentiveApplicationId
+                @event.AccountLegalEntityId,
+                MapForIncentive(@event.Apprenticeships)
                 );
 
             return _commandPublisher.Publish(command);
         }
+
+        private static List<CreateIncentiveCommand.IncentiveApprenticeship> MapForIncentive(IEnumerable<ApprenticeshipModel> apprenticeships)
+        {
+            return apprenticeships.Select(a => new CreateIncentiveCommand.IncentiveApprenticeship(
```

Comment:
This is where the apprenticeships for the submitted application should be looked up and a command per apprenticeship created.  Then this has the single responsibility of converting a submitted event into a CreateIncentiveCommand.

Reply:
Which bounded context does this class form part of?  Hard to tell from our structure as everything is intermingled.

Reply:
The folder structure gives the scope.  So this is in the IncentiveApplications folder so its part of that bounded context.  We could move into separate projects to make clearer perhaps?